### PR TITLE
Add a blank line before day line on khal list / calendar. Fixes #968

### LINKED ISSUE #968: Spacing between days in event view
The example image on the github project page shows a blank line in the event view between "Today" and "Tomorrow" creating a grouping of sorts. It's not possible to achieve that with khal version 0.10.2.
What I get:
```
$ khal calendar 
    må ti on to fr lö sö     Today, 2020-10-20
okt 28 29 30  1  2  3  4     09:30-18:00 Foo
 41  5  6  7  8  9 10 11     Tomorrow, 2020-10-21
 42 12 13 14 15 16 17 18      Test Event :: Desc!
 43 19 20 21 22 23 24 25     
nov 26 27 28 29 30 31  1     
 45  2  3  4  5  6  7  8     
 46  9 10 11 12 13 14 15     
 47 16 17 18 19 20 21 22     
 48 23 24 25 26 27 28 29     
dec 30  1  2  3  4  5  6     
 50  7  8  9 10 11 12 13     
 51 14 15 16 17 18 19 20     
 52 21 22 23 24 25 26 27
```

What I want (and see on the github page):
```
$ khal calendar 
    må ti on to fr lö sö     Today, 2020-10-20
okt 28 29 30  1  2  3  4     09:30-18:00 Foo
 41  5  6  7  8  9 10 11
 42 12 13 14 15 16 17 18     Tomorrow, 2020-10-21
 43 19 20 21 22 23 24 25      Test Event :: Desc!     
nov 26 27 28 29 30 31  1     
 45  2  3  4  5  6  7  8     
 46  9 10 11 12 13 14 15     
 47 16 17 18 19 20 21 22     
 48 23 24 25 26 27 28 29     
dec 30  1  2  3  4  5  6     
 50  7  8  9 10 11 12 13     
 51 14 15 16 17 18 19 20     
 52 21 22 23 24 25 26 27
```

### PR DESCRIPTION
**Pull request by ephase**
Add a blank line before the day / date line in `khal list` and `khal calendar` except for the first one. Update test in `cli_test.py`.

List is more cleared with this in my opinion.

### CODE REVIEW

**Comment by d7415** [comment]
I'm not sure about changing the default, especially without a way of putting it back. On the other hand, it would match the screenshot in the readme...

**Comment by ephase** [comment]
@d7415 I can add a parameter in config file like `blank_line_before_day_line` in `[default]` section.

**Comment by ephase** [comment]
I made some modifications and add an option for the blank line.

**Comment by ephase** [comment]
@fredthomsen I investigated a little more today and found that when the search window appear, the 'normal' `EventColumn`lost focus, and there is no more `self.focus_date`. **But** when you press the left key, 'CalendarWidget` have focus, navigate a little in this widget create a new event then the date used in the create event window is right.

For me, the solution in my PR seems to be right, what do you think @d7415 ?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -18,6 +18,8 @@ not released
 * FIX Alarms without descriptions no longer crash `ikhal`
 * FIX Display all-day events at the top of the day in `ikhal`
 * FIX Keybindings in empty search results no longer crash `ikhal`
+* NEW Possibility to adda bank line before day in `khal` with
```
Comment (d7415) [inline CHANGELOG.rst:21]:
```suggestion
* NEW Possibility to add a blank line before day in `khal` with
```

File: khal/settings/khal.spec [hunk 2]
```diff
@@ -230,6 +230,9 @@ event_view_weighting = integer(default=1)
 # Set to true to always show the event view window when looking at the event list
 event_view_always_visible = boolean(default=False)
 
+# Add a blank line before the name of the day (khal only)
```
Comment (d7415) [inline khal/settings/khal.spec:233]:
```suggestion
# Add a blank line before the name of the day (khal only)
```

**Review by d7415 — APPROVED** [review]
> @fredthomsen I investigated a little more today and found that when the search window appear, the 'normal' `EventColumn`lost focus, and there is no more `self.focus_date`. **But** when you press the left key, 'CalendarWidget` have focus, navigate a little in this widget create a new event then the date used in the create event window is right.
> 
> For me, the solution in my PR seems to be right, what do you think @d7415 ?

I'm inclined to agree, but I think that was meant for #998?

As for this PR, it looks good and appears to work as advertised - Just a typo and a trailing space in comments.

**Comment by ephase** [comment]
@d7415 you're right for the message, sorry for the mistake...

**Pull request merged**